### DISCUSSION TITLE
Job is submitted but not running

### DISCUSSION THREAD

**Discussion by Jinnan-Guo** [NVIDIA/NVFlare#1213] [Q&A]
I recently submit a job with (maybe) correct meta.json configurations, but job scheduler cannot schedule it to run

So I start wondering why job scheduler cannot approve this job. I am also wondering if there is a cmd or api for admin to check why a job is not approved to run. From the code NVFlare will raise value error if a job cannot pass the validation. May I know where can I get these error to debug? Many thanks!

**Comment by YuanTingHsieh** [comment]
Hi @Jinnan-Guo thanks for the interest.

Were you able to run the NVFlare examples?

If so, can your share your meta.json, config_fed_server.json and config_fed_client.json?
You can also try to use "nvflare simulator" to run your custom apps.

**Comment by YuanTingHsieh** [comment]
@Jinnan-Guo I'll close this due to inactivity, feel free to re-open if you have more questions.
